README.md

```diff
@@ -27,5 +27,23 @@ fruit.getModule("pineap").enable()
 <pre>
 fruit.getModule("pineap").deauth('6e:74:64:69:63:6b', ['73:65:62:6b:69:6e', '6e:65:73:67:69:61'], 5, 1)
 </pre>
-
+##### Get SSID Pool
+<pre>
+p = fruit.getModule("pineap").getSSIDPool()
+</pre>
+Returns a dict. The pool is on the key "ssidPool" separated by newlines. To get a quick list, do the following:
+<pre>
+ssids = p['ssidPool'].split('\n')
```

Comment:
shouldn't p['ssidPool'] return a list instead of a '\n'-joined string?

Reply:
Yea probably but that's what's returned from the API raw. I've been using logging.getPineapLog() instead because it comes back structured with clients in json. I'll be publishing some more stuff and rewriting a bunch more of this library in my own repo as I get to it. I have to modify this PR most likely, as it wasn't backwards compatible with older firmware. The pineapple.api in mine is now but before submitting more PRs, I wanted to rewrite a few more things.